src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -1993,16 +1993,21 @@ internal static string ConcatenateStringPathArguments(CommandElementAst stringAs
             CompletionContext context,
             Dictionary<string, AstParameterArgumentPair> boundArguments = null)
         {
-            if (string.IsNullOrEmpty(commandName))
-            {
-                return;
-            }
-
             var parameterName = parameter.Name;
-            var customCompleter = GetCustomArgumentCompleter(
-                    "CustomArgumentCompleters",
-                    new[] { commandName + ":" + parameterName, parameterName },
-                    context);
+            ScriptBlock customCompleter;
```

Comment:
do you need this separated out? It use to just be `var customCompleter`

Reply:
Agree that the declaration should be together with the assignment. But better use `ScirptBlock` instead of `var`.

Reply:
Addressed.